widget/embedded/src/store/quote.ts

```diff
@@ -74,40 +86,72 @@ export const useQuoteStore = createSelectors(
       outputUsdValue: new BigNumber(0),
       toBlockchain: null,
       toToken: null,
-      quote: null,
+      refetchQuote: true,
+      sortStrategy: 'Maximum Return',
+      selectedQuote: null,
+      quotes: null,
+      selectedTag: null,
       quoteWalletsConfirmed: false,
       selectedWallets: [],
       customDestination: '',
       quoteWarningsConfirmed: false,
-      setQuote: (quote) =>
+      setSelectedQuote: (quote) =>
         set((state) => {
           let outputAmount: BigNumber | null = null;
           let outputUsdValue: BigNumber = ZERO;
           if (!isPositiveNumber(state.inputAmount)) {
             return {};
           }
+          const selectedQuote =
+            quote && 'result' in quote ? quote.result : quote;
           if (!!quote) {
-            outputAmount = !!quote.result?.outputAmount
-              ? new BigNumber(quote.result?.outputAmount)
+            outputAmount = !!selectedQuote?.outputAmount
+              ? new BigNumber(selectedQuote?.outputAmount)
               : null;
             outputUsdValue = calcOutputUsdValue(
-              quote.result?.outputAmount,
-              getQuoteToTokenUsdPrice(quote) || state.toToken?.usdPrice
+              selectedQuote?.outputAmount,
+              getQuoteToTokenUsdPrice(selectedQuote) || state.toToken?.usdPrice
             );
           }
           return {
-            quote: quote,
+            selectedQuote: quote,
             ...(!!quote && {
               outputAmount,
               outputUsdValue,
             }),
           };
         }),
+      setQuotes: (quotes) =>
+        set(() => {
+          return {
+            quotes,
+          };
+        }),
+      setSelectedTag: (selectedTag) =>
+        set(() => {
+          return {
+            selectedTag,
+          };
+        }),
+      setSortStrategy: (sortStrategy) =>
+        set(() => {
+          return {
+            sortStrategy,
+          };
+        }),
+      setRefetchQuote: (refetchQuote) =>
+        set(() => {
+          return {
+            refetchQuote,
+          };
+        }),
```

Comment:
You can use this :
set({property: value})

Reply:
fixed